backend/src/core/config.ts

```diff
@@ -51,6 +51,11 @@ const config = convict({
       default:  'ap-northeast-1',
       env: 'AWS_REGION',
     },
+    awsEndpoint: {
```

Comment:
I was reading the docs for secrets manager, and cloudwatch. The endpoint property is expected to be a string like 'https://{service}.{region}.amazonaws.com'. Since I'm not familiar with localstack- does this mean we should be adding multiple, specific `awsEndpoint`  for each service here? 

References:
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SecretsManager.html#constructor-property
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CloudWatchLogs.html#constructor-property
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor-property

Reply:
Not with localstack, or at least, not with _current_ versions of localstack; one endpoint will offer all the services.